CONTRIBUTING.md

```diff
@@ -131,6 +131,20 @@ cargo check --all-features
 cargo test --all-features
 ```
 
+Clippy must be run using the MSRV, so Tokio can avoid having to `#[allow]` new
+lints whose fixes would be incompatible with the current MSRV:
+
+<!--
+When updating this, also update:
+- .github/workflows/ci.yml
+- README.md
+- tokio/README.md
+-->
```

Comment:
it would be really nice to have a script for this or something, or some way to to have a single authoritative definition of the MSRV that is referenced elsewhere. there are now several places where the MSRV will need to be updated.

obviously, this would be best left for a separate PR, but i thought it was worth noting.